README.md

```diff
@@ -129,16 +137,17 @@ Tauri is a system composed of a number of moving pieces:
 - github actions for CI and CD
 - discord for discussions
 - netlify-hosted documentation website
+- digital ocean meilisearch instance
 
 ### Major Runtimes
 - node.js for running the CLI (deno and pure rust are on the roadmap)
 - cargo for testing, running the dev service, building binaries and as the runtime harness for the webview
 
 ### Major Languages
-- typescript for the CLI
-- javascript for the JS-side of the API
-- rust for bindings, rust side of the API, harnesses
-- C / obj-c for interfacing with the webviews
+- Rust for the CLI
+- EcmaScript bindings to the Rust API, written in typescript
```

Comment:
Should we say JavaScript instead of EcmaScript here to make it more clear for beginners?